.cirrus.yml

```diff
@@ -234,6 +235,7 @@ integration_task:
             DISTRO_NV: "${DEBIAN_NAME}"
             IMAGE_NAME: "${DEBIAN_CACHE_IMAGE_NAME}"
             STORAGE_DRIVER: 'overlay'
+            CI_DESIRED_RUNTIME: runc
```

Comment:
Not an issue for now, but can/should we change this to crun at some point?

Reply:
In Podman world all tests on debian are using runc and crun on Fedora,  On buildah we are running all tests with crun.